github/actions_cache_test.go

```diff
@@ -590,7 +590,11 @@ func TestActionsCacheList_Marshal(t *testing.T) {
 
 func TestActionsCacheUsage_Marshal(t *testing.T) {
 	t.Parallel()
-	testJSONMarshal(t, &ActionsCacheUsage{}, "{}")
+	testJSONMarshal(t, &ActionsCacheUsage{}, `{
+		"active_caches_count": 0,
+		"active_caches_size_in_bytes": 0,
+		"full_name": ""
+	}`)
```

Comment:
there are several tests that were expecting an empty json (`{}`) before this change but have properties/fields with zero values defined (like this case here).

Is the idea to have these fields really empty or is it ok to have these values defined like this?

Reply:
It looks to me like these values are actually showing up when `omitempty` is not used, as in this case.
So I'm thinking that this PR provides a realistic view of what is actually being passed when a zero-value struct is being sent across the wire, which I think is a good thing, as it demonstrates what is actually happening under the hood.

Reply:
Would it be valid for me to open a new PR with these changes (applying `omitempty`) and merge it before this one? That way we don't have to "do" and then "undo" the changes into some test cases. I can do that in this PR too.

If it's viable, what approach would you suggest?

Reply:
Hmm... I'm extremely hesitant to add/remove `omitempty` unless there is a great reason to do so.

Over the history of this repo, I would venture that the majority of "bugs" were caused by the presence or absence of `omitempty` and how it interacted with the GitHub v3 API.

So I would say "no" to another PR that changes `omitempty` for the sake of tests.

But maybe I'm misunderstanding your question.

Reply:
Got it. That's exactly what it's about.
Thanks for sharing the context.